webrtc/interfaces.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+<html>
+  <head>
+    <title>"WebRTC IDL Tests</title>
```

Comment:
Mm not sure if removing even the `<html>` and `<body>` tags is a good idea. Do you have an example of other WG tests omitting them? I'd prefer to keep them at least.